### PR TITLE
[tools-v2] add bs check server based on pr2463

### PR DESCRIPTION
**Pull request by caoxianfei1**
### What problem does this PR solve?

This pr is implemented based on https://github.com/opencurve/curve/pull/2463, add bs check server command

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by caoxianfei1** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by caoxianfei1** [comment]
@montaguelhz PTAL

File: tools-v2/pkg/cli/command/curvebs/check/server/server.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+		row[cobrautil.ROW_UNHEALTHY_COPYSET] = strconv.FormatUint(uint64(unhealthy), 10)
+		row[cobrautil.ROW_UNHEALTHY_COPYSET_RATIO] = fmt.Sprintf("%v%%", (unhealthy/total)*100)
+		sCmd.rows = append(sCmd.rows, row)
+	}
+
+	list := cobrautil.ListMap2ListSortByKeys(sCmd.rows, sCmd.Header, []string{
+		cobrautil.ROW_SERVER,
+	})
+	sCmd.TableNew.AppendBulk(list)
+	sCmd.Result = sCmd.rows
```
[the remark this answers is not in the document]
Comment (caoxianfei1) [reply tools-v2/pkg/cli/command/curvebs/check/server/server.go:153]: fixed

File: tools-v2/pkg/cli/command/curvebs/check/server/server.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	checkcopyset "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/check/copyset"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/chunkserver"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+const (
+	serverExample = `$ curve bs check server`
```
[the remark this answers is not in the document]
Comment (caoxianfei1) [reply tools-v2/pkg/cli/command/curvebs/check/server/server.go:40]: fixed

**Review by Cyber-SiKu — APPROVED** [review]

**Comment by montaguelhz** [comment]
LGTM

**Review by montaguelhz — APPROVED** [review]

**Comment by caoxianfei1** [comment]
cicheck

**Pull request merged**